### ISSUE TITLE
No hover state on links on About Us page

### ISSUE THREAD

**Issue by alisonrae** [PublicisSapient/accessible-ecommerce-demo#261]
On the About Us page, add an underline when the user hovers over the links on the left column. Right now the only hover indicator that these are links is that the cursor changes. We need to make it more obvious.

We also use color alone to indicate hover on the white buttons with a black outline that turn grey on hover. Let's do a sweep and give everything an underline on hover.

**Comment by liskulic** [comment]
Updated. Pending on PR approval and merge/deploy.
